lighthouse-core/test/fixtures/fraggle-rock/reports/sample-flow-result.json

```diff
@@ -9092,11 +9092,9 @@
             "id": "experimental-interaction-to-next-paint",
             "title": "Interaction to Next Paint",
             "description": "Interaction to Next Paint measures page responsiveness, how long it takes the page to visibly respond to user input. [Learn more](https://web.dev/inp/).",
-            "score": 1,
-            "scoreDisplayMode": "numeric",
-            "numericValue": 60,
-            "numericUnit": "millisecond",
-            "displayValue": "60 ms"
+            "score": null,
+            "scoreDisplayMode": "error",
+            "errorMessage": "This version of Chrome is too old to support 'detailed EventTiming trace events'. Use a newer version to see full results."
```

Comment:
Could you rebaseline the flow artifacts? Should be able to use `--rebaseline-artifacts Trace` but there is no flag to modify a specific flow step yet.

Reply:
> Could you rebaseline the flow artifacts? Should be able to use `--rebaseline-artifacts Trace` but there is no flag to modify a specific flow step yet.

> ```
> TypeError: Cannot set property startTime of #<PerformanceEntry> which has only a getter
>     at Object.normalizeTimingEntries (~/lighthouse/lighthouse-core/lib/asset-saver.js:324:22)
>     at rebaselineArtifacts (~/lighthouse/lighthouse-core/scripts/update-flow-fixtures.js:114:16)
>     at async ~/lighthouse/lighthouse-core/scripts/update-flow-fixtures.js:157:7
> ```

I'll add it to the TODO list in #13916 and we can fix/update in a follow up.